cppcache/src/CacheImpl.cpp

```diff
@@ -744,4 +742,21 @@ CacheImpl::getMemberListForVersionStamp() {
   return *versionStampMemIdList;
 }
 
+std::unique_ptr<DataInput> CacheImpl::createDataInput(const uint8_t* buffer,
+                                                      size_t len) const {
+  auto poolName = getPoolManager().getDefaultPool()->getName();
+  return std::unique_ptr<DataInput>(new DataInput(buffer, len, this, poolName));
```

Comment:
I bet the `std::`committee looks back and laments how `make_unique` didn't get into ++11